docker_interop/quorum_genesis.json

```diff
@@ -0,0 +1,36 @@
+{
+  "config": {
+    "chainId": 2017,
+    "homesteadBlock": 0,
+    "istanbul": {
+      "epoch": 30000,
```

Comment:
can we specify the block period explicitly here and in besu config so we know that they match. think it is actually a command line option for quorum.